library/src/main/java/com/datatorrent/lib/db/jdbc/JdbcPOJOInputOperator.java

```diff
@@ -632,5 +634,26 @@ public void setMysqlSyntax(boolean mysqlSyntax)
     this.mysqlSyntax = mysqlSyntax;
   }
 
+  /**
+   * Function to initialize the list of {@link FieldInfo} externally from configuration/properties file
+   * @param index
```

Comment:
Can you please describe what the parameters (index, value) are supposed to be?
Also please give an example of how the configuration would look like in case the user configures it from a properties file.